app/views/hosts/console/spice.html.erb

```diff
@@ -2,16 +2,16 @@
 <% title "#{@console[:name]}" %>
 <%= title_actions(
         button_group(link_to(_("Ctrl-Alt-Del"), "#", :id => "sendCtrlAltDelButton", :onclick => 'sendCtrlAltDel()', :class => "btn btn-default"),
-                     link_to(_("Troubleshooting"), "http://theforeman.org/manuals/#{SETTINGS[:version].short}/index.html#7.1NoVNC",
-                       :id => "troubleshootingButton", :class => "btn btn-default", :target => "_blank"),
                      if supports_spice_xpi?
                        link_to(_("New window"), "#", :id => "disconnect", :class => "btn btn-default", :onclick => 'connectXPI()', :title => _("Open Spice in a new window"))
                      end,
                      if @host
                        link_to_if_authorized(_("Back to host"), hash_for_host_path(:id => @host), :title => _("Back to host"), :class => 'btn btn-default')
                      end
-        )) %>
-
+                   ),
+                   documentation_button("7.1NoVNC",
+                     {:id => "troubleshootingButton", :class => "btn btn-default"})
```

Comment:
Nitpick - I think the indentation here is a bit off, the documentation_button should be on the same level as the button group iiuc